src/frontend/Sider/Behandling/faner.tsx

```diff
@@ -9,59 +9,66 @@ import {
 
 import Brev from './Brev/Brev';
 import Inngangsvilkår from './Inngangsvilkår/Inngangsvilkår';
+import Stønadsvilkår from './Stønadsvilkår/Stønadsvilkår';
 import VedtakOgBeregningBarnetilsyn from './VedtakOgBeregning/Barnetilsyn/VedtakOgBeregningBarnetilsyn';
+import { Stønadstype } from '../../typer/behandling/behandlingTema';
 
 export type FanerMedRouter = {
-    navn: FaneNavn;
+    navn: string;
```

Comment:
Hvorfor endre denne?

Reply:
Fordi navnet "pass barn" ikke lenger lå i fanenavn, men endret nå så varierende navn for stønadsvilkår ble eget enum